### PR TITLE
Filezilla cheat sheet

### PR DESCRIPTION
**Pull request by codenirvana**
**What does your Instant Answer do?**
A cheat sheet that lists useful Filezilla keyboard shortcuts.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It summaries all the most commonly used shortcuts so you don't have to go through several links/sources.

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Filezilla](https://wiki.filezilla-project.org/Keyboard_shortcuts) official wiki

**Why did you choose this data source?**
They list all the most commonly used Filezilla keyboard shortcuts.

**What are some example queries that trigger this Instant Answer?**
- Filezilla cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Web developers

**Which existing Instant Answers will this one supercede/overlap with?**
None

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot
https://guides.github.com/features/mastering-markdown/

-----
IA Page: https://duck.co/ia/view/filezilla_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/filezilla.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "name": "Filezilla",
+    "description": "Filezilla Keyboard Shortcuts ",
+    "metadata": {
+        "sourceName": "Filezilla",
+        "sourceUrl": "https://wiki.filezilla-project.org/Keyboard_shortcuts"
+    },
+    "template_type": "keyboard",
+    "section_order": ["General"],
+    "sections": {
+        "General": [{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/filezilla.json]: Let's break this up into a few groups. Ideally by grouping them into meaningful sections. Do these commands fall under file-menu headings?

File: share/goodie/cheat_sheets/json/filezilla.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
+    "id": "filezilla_cheat_sheet",
+    "name": "Filezilla",
+    "description": "Filezilla Keyboard Shortcuts ",
+    "metadata": {
+        "sourceName": "Filezilla",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/filezilla.json]: nit: FileZilla (casing)

File: share/goodie/cheat_sheets/json/filezilla.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{
+    "id": "filezilla_cheat_sheet",
+    "name": "Filezilla",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/filezilla.json]: Casing here too: FileZilla

**Comment by codenirvana** [comment]
@moollaza check now.

File: share/goodie/cheat_sheets/json/filezilla.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
+{
+    "id": "filezilla_cheat_sheet",
+    "name": "FileZilla",
+    "description": "FileZilla Keyboard Shortcuts ",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/filezilla.json]: We can remove this as it's redundant. OR, we should provide a better description that describes what the cheat sheet is doing

**Comment by codenirvana** [comment]
@moollaza Description removed.

**Comment by moollaza** [comment]
@codenirvana Thanks! This LGTM :+1:

**Pull request merged**